{"name": "test_create_new_user", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/Users/juan/Desktop/users.json'", "trace": "def test_create_new_user():\n        #Read input json file\n>       file = open('/Users/juan/Desktop/users.json','r')\nE       FileNotFoundError: [Errno 2] No such file or directory: '/Users/juan/Desktop/users.json'\n\nTestCases/test_CreateUser.py:10: FileNotFoundError"}, "start": 1682543135731, "stop": 1682543135731, "uuid": "1246ada8-fd2c-48c3-ad69-e1f4193cc1f7", "historyId": "944868ec838ced5bc7f5fd66cbb3d85d", "testCaseId": "32d8263403fc2a680b700f6c1187dd8c", "fullName": "TestCases.test_CreateUser#test_create_new_user", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_CreateUser"}, {"name": "host", "value": "Juans-Mac-mini.local"}, {"name": "thread", "value": "6732-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_CreateUser"}]}